src/api_codes.js

```diff
@@ -147,6 +153,7 @@ export const statusWideangleCmd = 10217;
 // goto
 export const calibrateGotoCmd = 11205;
 export const startGotoCmd = 11203;
+export const stopGotoCmd = 11204;
```

Comment:
Cool that you added stop goto. How did you get the code for stopGoto? I didn't see it in the 1.0 API docs.

Reply:
see the request on the fly by using wireshark